ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -4,6 +4,44 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+function populate_artifact_dir() {
+  set +e
+  echo "Copying log bundle..."
+  cp "${dir}"/log-bundle-*.tar.gz "${ARTIFACT_DIR}/" 2>/dev/null
+  echo "Removing REDACTED info from log..."
+  sed '
+    s/password: .*/password: REDACTED/;
+    s/X-Auth-Token.*/X-Auth-Token REDACTED/;
+    s/UserData:.*,/UserData: REDACTED,/;
+    ' "${dir}/.openshift_install.log" > "${ARTIFACT_DIR}/.openshift_install.log"
+}
+
+function prepare_next_steps() {
+  rv=$?
```

Comment:
What will this value be if `prepare_next_steps` is called in response to an external interrupt rather than a failed command? Do we care that we may touch the success file in this case instead of the failure? I think it may be better to leave the logic for touching the success or failure file out of this function and done only using the return value from the `openshift-install create cluster` call.

Reply:
> What will this value be if prepare_next_steps is called in response to an external interrupt rather than a failed command?
something other than 0, never checked what is the actual value
> Do we care that we may touch the success file in this case instead of the failure?
I don't think that will happen because that will send an event to the ovirt CI UI with success (next steps), but anyway it doesn't have a real impact
> I think it may be better to leave the logic for touching the success or failure file out of this function and done only using the return value from the openshift-install create cluster call
the thing is that if we would do it outside the trap then in case of an interrupt we will not create the file... I'm ok with that it will just require me to change the logic of the send event step

Reply:
Edited @staebler